osf/migrations/0135_user_settings_waffles.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-10-03 14:20
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from osf.utils.migrations import AddWaffleFlags
+
+
+USER_SETTINGS_FLAGS = [
+    'ember_user_settings_account_page',
+    'ember_user_settings_addons_page',
+    'ember_user_settings_notifications_page',
+    'ember_user_settings_apps_page',
+    'ember_user_settings_tokens_page',
+]
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0134_abstractnode_custom_citation'),
+    ]
+
+    operations = [
+        AddWaffleFlags(USER_SETTINGS_FLAGS),
```

Comment:
Lately, we haven't been using migrations to add new waffle flags. It's unnecessary, since we manage flags through the admin app anyway.

Reply:
Discussed on Flowdock:

From @aaxelb :

> I think migrations are a good fit for manage flags' existence. After we introduce a flag, it exists everywhere and is easy to turn on without having to copy-paste the name from a pull request (which I think is how embosf flags get made on stagings/prod). When we stop using a flag, it stops existing on every environment, even locally.
> ...so there are no stale flags, and no missing flags

I agree with this, so let's stick with the approach in this PR.